src/parser/classes/CompactVideo.ts

```diff
@@ -39,6 +40,7 @@ class CompactVideo extends YTNode {
     this.view_count = new Text(data.viewCountText);
     this.short_view_count = new Text(data.shortViewCountText);
     this.published = new Text(data.publishedTimeText);
+    this.is_live = data.badges?.some((badge: any) => badge.metadataBadgeRenderer.style == 'BADGE_STYLE_TYPE_LIVE_NOW') || false;
```

Comment:
Please use 3 equals signs here, as you want to compare the exact values.
```suggestion
    this.is_live = data.badges?.some((badge: any) => badge.metadataBadgeRenderer.style === 'BADGE_STYLE_TYPE_LIVE_NOW') || false;
```